content/blog/2021-11-05.md

```diff
@@ -0,0 +1,206 @@
+---
+title: The XMPP Newsletter October 2021
+author: emus
+date: 2021-11-05
+categories: ['Newsletter']
+---
+
+Welcome to the XMPP Newsletter covering the month of October 2021.
+
+Many projects and their efforts in the XMPP community are a result of people’s voluntary work. If you are happy with the services and software you may be using, especially throughout the current situation, please consider to say thanks or help these projects!
+
+[Read this Newsletter via our RSS Feed](https://xmpp.org/feeds/all.atom.xml)!
+
+Interested in supporting the Newsletter team? Read more at the bottom.
+
+Other than that - enjoy reading!
+
+### Newsletter translations
+
+Translations of the XMPP Newsletter will be released here (with some delay):
+
+- The French translation appears at [jabberfr.org](https://news.jabberfr.org/category/newsletter/) and [linuxfr.org](https://linuxfr.org/tags/xmpp/public)
+- The German translation appears at [anoxinon.de](https://anoxinon.de/blog/)
+- The Italian translation appears at [NicFab.it](https://www.nicfab.it/)
+
+Many thanks to the translators and their work! This is a great help to spread the news! Please join them in their work or start over with another language!
+
+## XSF Announcements
+
+- The XSF members have to elect the upcoming XSF Board and Council soon. When you are interested in running for XSF Board (business related) or XSF Council (technical related) for the 2021 - 2022 term, please [create an appropriate Wiki page by November 7th, 2021 and link it from here](https://wiki.xmpp.org/web/Board_and_Council_Elections_2021)
+
+- The XSF offers [fiscal hosting](https://xmpp.org/community/fiscalhost/) for XMPP projects now! Please apply via Open Collective. For more information, see the [announcement blog post](https://xmpp.org/2021/09/the-xsf-as-a-fiscal-host/).
+
+    - Moreover, the XSF has accepted its first project to the fiscal hosting program! A big welcome to the [MAM Plugin for XMPP.js](https://gitlab.com/deblounge/xmppjs-mam-plugin) (note that this is unaffiliated with the upstream [xmpp.js](https://github.com/xmppjs/xmpp.js) project)! They will be working on adding MAM support to xmpp.js with the eventual goal of adding it to the Matrix Bifrost bridge, allowing XMPP users to fetch history from their favorite Matrix channels. You can [donate to the effort here](https://opencollective.com/mam-plugin-for-xmppjs).
+
+- The XSF is planning to participate the Google Summer of Code 2022 (GSoC). If you are interested in participating as a student, mentor or as project in general please [add your ideas and reach out to us](https://wiki.xmpp.org/web/Google_Summer_of_Code_2022)!
+
+- The blog and newsletter XSF pages offer now multiple languages. We are happy for volunteers to support translating!
```

Comment:
Blog and newsletter pages at [xmpp.org/blog](https://xmpp.org/blog) now support multiple languages.